tests/test-notebooks/test_notebooks.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+from nbconvert.preprocessors.execute import CellExecutionError
+
+import glob
+import traceback
+import unittest
+import sys
+import time
+import os
+
+
+class TestNotebooks(unittest.TestCase):
```

Comment:
i thought about this. we don't depend on any specific tensorflow functionality here (like having access to a session). this is why i kept it as a `unittest`